src/index.njk

```diff
@@ -61,9 +62,12 @@ sitemapPriority: 1.0
         </div>
         <div class="flex flex-col-reverse sm:flex-row-reverse justify-between gap-3 sm:gap-12">
             <div class="text-center sm:text-left pt-4">
-                <h3>Manage Remote Deployments</h3>
-                <p>Many organizations deploy Node-RED instances to remote servers or edge devices. FlowFuse automates this process by creating snapshots on Node-RED instances that can be 
-                deployed to multiple remote targets. It also allows for rollback in cases where something might not have gone correctly.</p>
+                <h3>Capture edge data from any protocol</h3>
```

Comment:
Competition seems to love edge, instead of remote. To be frank, remote feels distant and something I don't intuitively care about either. Like a remote friend. Edge is better IMO